src/main/scala/ru/d4team/chat/models/message/MessageReport.scala

```diff
@@ -0,0 +1,18 @@
+package ru.d4team.chat.models.message
+
+import io.scalaland.chimney.dsl.TransformerOps
+
+import java.time.Instant
+import java.util.UUID
+
+case class MessageReport(
```

Comment:
Do you mean `MessageResponse`?

Reply:
@AlekseiLitkovetc 
I thought about it.  Yesterday it seemed more logical to me, but mb it's better to use the response.

Reply:
Ok, fixed